modules/exploits/windows/local/cve_2020_1170_windows_defender.rb

```diff
@@ -218,23 +215,20 @@ def check
     gem_version_mpfile = Gem::Version.new(mprun_file_version.join('.')) # Taken from https://stackoverflow.com/questions/4018689/ruby-combining-an-array-into-one-string
     print_status("Version #{gem_version_mpfile} of MpCmdRun.exe detected on target!")
 
-    begin
-      mpcomputerstatus_result = cmd_exec('powershell -c "Get-MpComputerStatus"', 360)
-    rescue Rex::TimeoutError
-      # Just attempt to continue, apparently @smcintyre-r7 found this was timing out for him on his systems, though I haven't been able to replicate this.
-    end
-
+    mpcomputerstatus_result = cmd_exec('powershell -c "Get-MpComputerStatus"', 360)
     if mpcomputerstatus_result.nil? || mpcomputerstatus_result.empty?
       fail_with(Failure::UnexpectedReply, 'Could not get the status of Windows Defender to determine if it is active or not!')
-      return
-    end
-
-    if mpcomputerstatus_result =~ /800106BA/ # Error code 0x800106BA occurs when another security product is installed according
+    elsif mpcomputerstatus_result =~ /800106BA/ # Error code 0x800106BA occurs when another security product is installed according
       # to https://answers.microsoft.com/en-us/windows/forum/windows_vista-security/windows-defender-error-0x800106ba/0e2f4e98-4e5f-4991-8994-0f8c80784547
       print_error('The target has an outdated version of MpCmdRun.exe installed but they have another AV installed.')
       return CheckCode::Safe
     end
 
+    mpsignature_result = cmd_exec('powershell -c "Update-MpSignature -UpdateSource InternalDefinitionUpdateServer"')
+    unless mpsignature_result =~ /80070490/
+      fail_with(Failure::UnexpectedReply, 'Failed to update the signature data in an unexpected way')
```

Comment:
Double negative in this sentence. I think this is meant to say `in the expected manner` or something similar.